solr-synonyms-api/app.py

```diff
@@ -10,4 +10,5 @@
 # Listen on all interfaces, and the catalog Python container expects the application to be on 8080.
 application = synonyms.create_application()
 if __name__ == '__main__':
-    application.run(host='0.0.0.0', port=8080, debug=True)
+    #application.run(host='0.0.0.0', port=8080, debug=True)
```

Comment:
Should this be changed? I don't think so because I'm guessing it was/is only for local, but I don't know what effect it might have on running in the pods.